package.json

```diff
@@ -25,12 +25,15 @@
   },
   "scripts": {
     "nx": "nx",
-    "start": "nx serve",
-    "build": "nx build",
+    "build": "nx run-many --target=build --all",
+    "build:all": "nx run-many --target=build --all",
     "build:actions": "yarn tsc --project ./.github/actions/tsconfig.json",
-    "test": "nx test",
-    "lint": "nx workspace-lint && nx lint",
-    "e2e": "nx e2e",
+    "test": "nx run nx-aws-cache:test",
```

Comment:
no project specifics in package.json please, should be generic... the same for `lint` and `e2e` task